mw-webapp/src/component/notice/Notice.tsx

```diff
@@ -0,0 +1,62 @@
+import {useEffect} from "react";
+import {Button} from "../button/Button";
+import {Title} from "../title/Title";
+import {HeadingLevel} from "../title/Title";
+import styles from "./Notice.module.scss";
+
+const MILLISECONDS = 1000;
```

Comment:
We no need this const in components. Id you will need it you can get it by props from logic layer. Am I wrong?

Reply:
We receive timer (seconds) inside the props, but setTimeout use miliseconds inside. We just convert it.